docker/Dockerfile

```diff
@@ -37,12 +37,13 @@ RUN virtualenv -p python2.7 pre141 \
 # setup pre venv
 # use older version of setuptools or things won't build
 RUN /rgtm/pre141/bin/pip install -U pip \
- && /rgtm/pre141/bin/pip install -U wheel setuptools==20.1.1 nose==1.3.7 \
+ && /rgtm/pre141/bin/pip install -U nose==1.3.7 \
+ && /rgtm/pre141/bin/pip install -U wheel setuptools==20.1.1 \
  && /rgtm/pre141/bin/pip install \
-      git+https://github.com/rosin-project/rosdistro_python@rosin_bughunt_0.2.20#egg=rosdistro \
+      git+https://github.com/rosin-project/rosdistro_python@rosin_bughunt_0.8.3_test#egg=rosdistro \
```

Comment:
I believe this is the wrong line to replace.

This again can't work: the `pre141` virtualenv *must* have this version of `rosdistro`.

did you mean to change the `post141` virtualenv?

Reply:
Oh yeah sorry, that was an unintended change. I updated the Dockerfile